Hover/Model/HoverConfiguration.swift

```diff
@@ -53,7 +53,7 @@ public struct HoverConfiguration {
                 color: HoverColor = .color(.blue),
                 size: CGFloat = 60.0,
                 imageSizeRatio: CGFloat = 0.4,
-                spacing: CGFloat = 12.0,
+                padding: UIEdgeInsets = .init(top: 12, left: 12, bottom: 12, right: 12),
```

Comment:
How does this change reflect on the UI? I'm assuming a padding of 12 everywhere is different than a spacing change 🤔 

Can you please share a screenshot of the before & after this change with the default values?

Reply:
![Simulator Screen Shot - iPhone 12 Pro - 2021-08-12 at 13 19 38](https://user-images.githubusercontent.com/987360/129188390-df324239-017f-4553-acc5-bdf23a83478e.png)
![Simulator Screen Shot - iPhone 12 Pro - 2021-08-12 at 13 19 30](https://user-images.githubusercontent.com/987360/129188398-a3e1ece2-bf05-4f2c-90d5-fcf4f3b92cf1.png)
![Simulator Screen Shot - iPhone 12 Pro - 2021-08-12 at 13 19 33](https://user-images.githubusercontent.com/987360/129188403-4dcf767e-de8e-4176-9d19-8c6f888927be.png)
![Simulator Screen Shot - iPhone 12 Pro - 2021-08-12 at 13 19 35](https://user-images.githubusercontent.com/987360/129188407-7c65f755-71ba-4f49-bd46-3c282315cb56.png)

Reply:
Looks good to me. Can you please update the `README.md` as well?

Reply:
I've missed that: which part?

Reply:
Ahh... There's nothing to update! - I thought our example had `spacing` but turns out it doesn't.